pkg/apis/operator/v1alpha1/accessor.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"knative.dev/pkg/apis"
+)
+
+// Common conditions.
+const (
+	InstallSucceeded     apis.ConditionType = "InstallSucceeded"
```

Comment:
I'd prefer for `DependenciesInstalled` to be common. It can be useful for eventing, too. It's essentially a no-op in this operator, but it's a nice hook used by downstream operators.

Reply:
SG, I'll consider that!

Reply:
One thing to consider on Conditions is that we generally try to pick the Type such that "True" conveys success and "False" conveys failure (vs. lack of completion).

@evankanderson has been working to try and upstream some of our thinking into the api-principles, but it's at least the rules we are trying to follow.